src/howler.core.js

```diff
@@ -406,7 +406,7 @@
       if (self.state === 'running' && self._suspendTimer) {
         clearTimeout(self._suspendTimer);
         self._suspendTimer = null;
-      } else if (self.state === 'suspended') {
+      } else if (self.state === 'suspended' || self.state === 'interrupted') {
```

Comment:
Seems that this check will never be true since `self.state` doesn't really depend on the AudioContext state (apart from initialisation inside `HowlerGlobal.prototype._setup`) and seems unlikely to ever have the `interrupted` value.
One way to fix this would be to check the state of AudioContext directly, but the downside is that the code gets more ugly and complicated:
```javascript
if (self.state === 'running' && self.ctx.state !== 'interrupted' && self._suspendTimer) {
  clearTimeout(self._suspendTimer);
  self._suspendTimer = null;
} else if (self.state === 'suspended' || self.state === 'running' && self.ctx.state === 'interrupted') {
```

Reply:
@nanek I reread the discussion and it turns out that's what you already proposed :)

Reply:
one more thing: in this case inside the `play` method the condition
```javascript
if (Howler.state === 'running') {
```
needs to be rewritten as
```javascript
if (Howler.state === 'running' && Howler.ctx.state !== 'interrupted') {
```
otherwise sounds will try to play before the context state is resumed

Reply:
added this fix to this PR https://github.com/goldfire/howler.js/pull/1106